### ISSUE TITLE
Go directly to the Sensei Home page in WP admin after completing the Sensei onboarding flow

### ISSUE THREAD

**Issue by donnapep** [Automattic/wp-calypso#86546]
### What

Once a user completes the [Sensei onboarding flow](https://wordpress.com/setup/sensei/senseiSetup), they should be taken to Sensei's Home page so that they can complete the onboarding checklist.

### Why

Currently, the user is taken to the My Home page in Calypso after completing the flow, where they are presented with a _Site setup_ onboarding checklist, not the Sensei-specific one. Calypso is also difficult to navigate, and it's easy to get lost when you're constantly switching in and out of Calypso vs. WP admin.

### How

Woo has already done this in their [ecommerce flow](https://wordpress.com/setup/ecommerce/storeProfiler). However, they take the user to a top-level Home page, whereas we'd like to take them to Sensei's Home page if possible (at least for now).

Dependent on #85270, as I don't think this should ship before that change.
